### PR TITLE
added result yaml and xml from example code

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

**Comment by WouterJ** [comment]
Shouldn't we also add an XML example? (we can help with creating the example if you need help)

**Comment by OskarStark** [comment]
Will do after holiday

Greetings from turkey ;)

Am Dienstag, 28. Juli 2015 schrieb Wouter J :

**Comment by OskarStark** [comment]
done @WouterJ 

btw. why is ```database```-key not used in the xml??

File: components/config/definition.rst [hunk 1]
```diff
@@ -395,6 +395,45 @@ with ``append()``::
...
+            memory:               false
+            parameters:           # Required
+    
+                # Prototype
+                name:
+                    value:                ~ # Required
+
+.. code-block:: xml
+
+    <config>
```
Comment (WouterJ) [inline components/config/definition.rst]: should be `<database>`

Comment (OskarStark) [inline components/config/definition.rst]: yes should, but i executed this code from the doc and this was the result.

bildschirmfoto 2015-08-10 um 18 24 29

Comment (OskarStark) [inline components/config/definition.rst]: any idea? @javiereguiluz @xabbuh

Comment (WouterJ) [inline components/config/definition.rst]: The root element is replaced by `<config>` when using configuration in a DI extension. However, this is about using config standalone, in which case the root element is just used (so `<database>`).

Comment (OskarStark) [inline components/config/definition.rst]: shall i add another key on top so he result uses database and the remove the config node?

Comment (WouterJ) [inline components/config/definition.rst]: no, we are talking about config usage standalone and not inside a DI extension in this article. so you should just replace the `<config>` element with `<database>`.

Comment (OskarStark) [inline components/config/definition.rst]: ah ok, sorry i missunderstood you

**Comment by OskarStark** [comment]
done @WouterJ

**Comment by OskarStark** [comment]
if i could do anything else, please let me know.

**Comment by xabbuh** [comment]
Is there a reason not to wrap the two code blocks in a configuration block?

**Comment by OskarStark** [comment]
i did this because of the different highlighting.

**Comment by xabbuh** [comment]
What do you mean with different highlighting? I mean we should not put both examples in one `code-block` directive, but we can wrap both code blocks in one configuration block like this:

```rst
.. configuration-block::

    .. code-block:: yaml

        # ...

    .. code-block:: xml

        <!-- ... -->
```

**Comment by OskarStark** [comment]
oh nice, will have a look at it (y)

**Comment by OskarStark** [comment]
updated @xabbuh

**Comment by xabbuh** [comment]
:+1:

**Comment by WouterJ** [comment]
:+1:

**Comment by WouterJ** [comment]
Thank you Oskar for your work on this one!

**Comment by OskarStark** [comment]
:+1: thank you for the review time ;-)

**Pull request closed without merging**